package.json

```diff
@@ -101,17 +101,17 @@
     "css.escape": "^1.5.1",
     "cssnano": "^7.0.6",
     "debug": "^4.4.0",
-    "eslint": "^9.17.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-import-resolver-typescript": "^3.7.0",
+    "eslint": "^9.20.1",
+    "eslint-config-prettier": "^10.0.1",
+    "eslint-import-resolver-typescript": "^3.8.0",
     "eslint-plugin-import-x": "^4.6.1",
-    "eslint-plugin-jest": "^28.10.0",
+    "eslint-plugin-jest": "^28.11.0",
     "eslint-plugin-unicorn": "^56.0.1",
     "express": "^4.21.2",
     "fast-plist": "^0.1.3",
     "get-stdin": "^9.0.0",
-    "globals": "^15.14.0",
-    "globby": "^14.0.2",
+    "globals": "^15.15.0",
+    "globby": "~14.0.2",
```

Comment:
Updating globby to v14.1.0 prevents tests from running due to dependency issues, as described in sindresorhus/globby#260. As a long-term solution, replacing it with [tinyglobby](https://github.com/SuperchupuDev/tinyglobby) could be considered. However, for now, we will address the issue by pinning the version to the specific minor.